roboteam_ai/src/analysis/DecisionMaker.cpp

```diff
@@ -14,11 +14,10 @@ namespace analysis {
 PlayStyle DecisionMaker::getRecommendedPlayStyle(BallPossession possession) {
     int amountOfRobots = world::world->getUs().size();
 
-    // subtract one robot if we need a keeper
-    if (robotDealer::RobotDealer::usesSeparateKeeper()) {
+    // subtract one robot if we have a keeper
+    if (robotDealer::RobotDealer::usesSeparateKeeper() && robotDealer::RobotDealer::keeperExistsInWorld()) {
```

Comment:
This means that if the keeper is lost in vision we think that we dont use a keeper anymore? Now sure how this will change behavior, in a good or bad way?